contracts/VoucherKernel.sol

```diff
@@ -1063,6 +1059,36 @@ contract VoucherKernel is IVoucherKernel, Ownable, Pausable, UsingHelpers {
         return promises[promiseKey].depositSe;
     }
 
+    /**
+     * @notice Get the holder of a supply
+     * @param _tokenIdSupply        ID of a promise which is mapped to the corresponding Promise
```

Comment:
Confused by the comment:
please reformulate "  @param _tokenIdSupply     ID of the order (aka VoucherSet) which is mapped to the corresponding Promise."
+ Please also correct it in the interface definition (IVoucherKernel.sol)